go/logger/standard.go

```diff
@@ -101,6 +101,7 @@ func (log *Standard) initLogging() {
 	// to point to the appropriate log file.
 	initLoggingBackendOnce.Do(func() {
 		logBackend := logging.NewLogBackend(os.Stderr, "", 0)
+		logBackend.Color = true
```

Comment:
is this still needed?  does this break logging for logfiles (by inserting ANSI codes?)